uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request.controller.js

```diff
@@ -826,6 +829,27 @@ define([
       vm.dismissModal();
     }
 
+    /**
+     * Stores the request from and to times original values
+     */
+    function rememberRequestOriginalTimes () {
+      ['from', 'to'].forEach(function (dateType) {
+        originalRequestTimes[dateType] =
+          moment(vm.request[dateType + '_date']).format('HH:mm');
+      });
+    }
+
+    /**
+     * Reverts the request from and to times back to the original values
+     */
+    function revertRequestOriginalTimes () {
```

Comment:
Also the PR is not updated with these changes of this commit. Please add them.

Reply:
Done.